common-util/src/main/java/org/sunbird/services/sso/impl/KeyCloakServiceImpl.java

```diff
@@ -134,50 +138,30 @@ private PublicKey toPublicKey(String publicKeyString) {
   }
 
   @Override
-  public Map<String, String> createUser(Map<String, Object> request) {
-    String userId = null;
-    String accessToken = null;
-    UserRepresentation user = createUserReqObj(request);
-    Response result = null;
+  public void updatePassword(String userId, String password) {
+
     try {
-      result = keycloak.realm(KeyCloakConnectionProvider.SSO_REALM).users().create(user);
+      String fedUserId = getFederatedUserId(userId);
```

Comment:
what will happen if fedUserId  is null or not found.

Reply:
Nothing , It will return empty object